clab/docker.go

```diff
@@ -79,33 +83,23 @@ func (c *cLab) CreateBridge(ctx context.Context) (err error) {
 		bridgeName = "br-" + netCreateResponse.ID[:12]
 	}
 	log.Debugf("container network %s : bridge name: %s", c.Conf.DockerInfo.Bridge, bridgeName)
-	log.Debug("Disable RPF check on the docker host part1")
-	var b []byte
-	b, err = exec.Command("sudo", "sysctl", "-w", "net.ipv4.conf.all.rp_filter=0").CombinedOutput()
-	if err != nil {
-		return fmt.Errorf("failed to disable Checksum Offloading on docker bridge: %v", err)
-	}
-	log.Debugf("%s", string(b))
-	//if err = sysctl.Set("net.ipv4.conf.default.rp_filter", "0"); err != nil {
-	//	return err
-	//}
-	log.Debug("Disable RPF check on the docker host part2")
```

Comment:
@henderiw what was the reason to disable RPF check twice (part1 and part2)? I removed the part2 as it seems unnecessary, unless it is needed for the reasons unknown to me.

Reply:
check #52

Reply:
Ah, sorry, my bad, I didn't notice it had different scopes in the commands (ipv4 vs default)

Both are needed to disable rpf on existing and new interfaces. I will adjust the code accordingly

Reply:
fixed in 7a6585e